docs/source/upcoming_release_notes/1199-cool-updates.rst

```diff
@@ -0,0 +1,37 @@
+1199 cool-updates
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- create a new sub module `pc.py`
```

Comment:
nitpick: I think this should be mentioned when talking about the PhotonCollimator device, since it isn't a new feature and isn't understandable without that context.

Reply:
moved doc to new devices section.